src/lib.rs

```diff
@@ -144,3 +145,7 @@ pub use gdk_ffi::enums::{
 pub use self::keys::{
     keyval_name
 };
+
+pub use cairo_interaction::{
+    ContextExt
```

Comment:
I think it's better to reexport in `prelude.rs` with the expectation that people will `use gdk::prelude::*;`